code-challenges.js

```diff
@@ -9,67 +9,186 @@
 // --------------------1) Create a function that determines if a temperature is below boiling point, at boiling point or above boiling point. Boiling point is 212 degrees Fahrenheit.
 // Use the test variables provided below. Expected outcome: "85 is below boiling point" "350 is above boiling point" "212 is at boiling point"
 
+//Pseudo Code
+//create a function named whatIsTemp that takes in 3 arguments of temperature: temp1, temp2, temp3 and returns 3 different outcomes: temperature below boiling point, at boiling point, or above boiling point.
+//if temp is below boiling point, return "85 is below boiling point"
+//if temp is above boiling point, return "350 is above boiling point"
+//if temp is at boiling point, return "212 is at boiling point"
+
 var temp1 = 85
 var temp2 = 350
 var temp3 = 212
 
+const whatIsTemp = (temp) => {
+    if (temp < 212) {
+        return `${temp} is below boiling point`
+    } else if (temp === 212) {
+        return `${temp} is at boiling point`
+    } else if (temp > 212) {
+        return `${temp} is above boiling point`
+    } else {
+        return "Not a temperature."
+    }
+}
+console.log(whatIsTemp(temp1))
+console.log(whatIsTemp(temp2))
+console.log(whatIsTemp(temp3))
 
 
 
 
 // --------------------2) Create a function that multiplies each number in the array by 5 using a for loop.
 // Use the test variable provided below. Expected outcome: [15, 35, 0, 30, -45] [10, 25, -40, 100, 160]
 
+//Pseudo code
+//write a function newNumbers that takes in an array of numbers
+//access each number in the array and multiply by 5
+//return a new array of each new product 
+
 var myForNumbers1 = [3, 7, 0, 6, -9]
 var myForNumbers2 = [2, 5, -8, 20, 32]
 
-
+const newNumbers = (numbers) => {
+    let newArray = []
+    for (i=0; i<numbers.length; i++){
+        newArray.push(numbers[i] * 5)
+    }
+    return newArray
+}
+console.log(newNumbers(myForNumbers1))
+console.log(newNumbers(myForNumbers2))
 
 
 
 // --------------------3) Create a function that multiplies each number in the array by 5 using map.
 // Use the test variable provided below. Expected outcome: [435, 30, 140, 80, 10] [40, -35, 0, 30, 60]
 
+//Pseudo code
+//write a function named mult5 that takes in an array of numbers
+//using map multiply each number in the array by 5 - map iterates through each value in an array
+//return a new array with expected outcome.
+
 var myMapNumbers1 = [87, 6, 28, 16, 2]
 var myMapNumbers2 = [8, -7, 0, 6, 12]
 
+// var mult5 = myMapNumbers1.map(value => {
+//     return value * 5
+// })
+// console.log(mult5)
 
+// var mult5 = myMapNumbers2.map(value => {
+//     return value * 5
+// })
+// console.log(mult5)
+
+// Realised this wasn't the right way to do it.
+
+const mult5 = (array) => {
+    return array.map(value => {
```

Comment:
You can also use the implicit return for simple return statements

```suggestion
    return array.map(value => value*5)
```